canopeum_frontend/src/components/analytics/OptionQuantitySelector.tsx

```diff
@@ -135,7 +135,7 @@ const OptionQuantitySelector = <TValue extends OptionQuantityValueType>(
       <ul className='list-group list-group-flush overflow-hidden mt-1'>
         {selected.map((optionQuantity, index) => (
           <li
-            className='list-group-item row d-flex justify-content-between'
+            className='list-group-item row d-flex align-items-center justify-content-between'
```

Comment:
Leave for a batch/site form fixes PR

```suggestion
            className='list-group-item row d-flex justify-content-between'
```